README.md

```diff
@@ -11,6 +11,20 @@ This API server enables audio transcription using the OpenAI Whisper models.
 
 # Build from source
 
+Before build make sure that **CGO_ENABLED** env is set to **1**
+
+```
+set CGO_ENABLED 1
+```
+
+or preferable set it parament. Then check it via
```

Comment:
maybe just make a note like "you can check this with this command"?

Reply:
Fixed